README.md

```diff
@@ -5,33 +5,23 @@ ffagc
 
 ## Initial Environment Setup
 
-build / install stuff I dunno who even uses rails any more:
+Set up the project with:
 
-    rake
-    bundle update
-    bundle install
+```sh
+  bundle install
+  bundle exec rake db:migrate RAILS_ENV=development
+  bundle exec rails server
+```
 
-You'll need to create config/secrets.yml with the form:
-
-    development:
-      secret_key_base: [[[LOTS OF HEX]]]
-
-    test:
-      secret_key_base: [[[LOTS OF HEX]]]
-
-    production:
-      secret_key_base: <%= ENV["SECRET_KEY_BASE"] %>
-
-Use 'rake secret' to generate the hex.  Don't check this file into source control!
+If you are running this project in production you'll need to set environment variables for secrets that include `ENV` in `config/secrets.yml`.
 
 ## Email Setup
 
-If you plan to send email, you'll need to create config/initializers/smtp_secret.rb
-with:
-
-    ENV['PASSWORD']="your smtp password"
+If using SMTP set the required ENV variables or fill in (and do not commit) `.env` like so:
 
-and edit config/environments/*.rb to include your smtp server information.
```

Comment:
Maybe it would be easier for you to look at the rendered Readme: https://github.com/FireflyArtsCollective/ffagc/blob/4946bc8a2167c8faf2960d7834c66d06bc2bed7a/README.md